src/oss-find-squats-lib/Mutators/RemovedCharacterMutator.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CST.OpenSource.FindSquats.Mutators
+{
+    /// <summary>
+    /// Generates mutations for removing a character in the string.
+    /// </summary>
+    public class RemovedCharacterMutator : IMutator
+    {
+        public MutatorType Kind { get; } = MutatorType.RemovedCharacter;
+
+        public IEnumerable<Mutation> Generate(string arg)
+        {
+            for (int i = 1; i < arg.Length; i++)
+            {
+                yield return new Mutation(
+                    mutated: $"{arg[..i]}{arg[(i + 1)..]}",
+                    original: arg,
+                    mutator: Kind,
+                    reason: $"Character Removed: {i}");
```

Comment:
```suggestion
                    reason: $"Character Removed: {arg[i]}");
```

Reply:
I actually think the index of the removed character is clearer than the content of the removed character in this case. But I could potentially include both.  

The Mutation object will contain both the original and the mutated version.